credentials/credentials.go

```diff
@@ -100,13 +120,17 @@ func Get(helper Helper, reader io.Reader, writer io.Writer) error {
 	}
 
 	serverURL := strings.TrimSpace(buffer.String())
+	if len(serverURL) == 0 {
```

Comment:
the `Get` operation on a credentials store only needs a server URL, username is not needed, as the operation is the following:
- I have a repository `foobar.io`, get me the credentials for this repo so I can authenticate on it